indico/modules/events/editing/client/js/components/ExtensionList.jsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {Dropdown} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+
+export default function ExtensionList({value, disabled, onChange, onFocus, onBlur}) {
+  const [options, setOptions] = useState(value.map(x => ({text: x, value: x})));
```

Comment:
We need the local state to handle the additions to the options

Reply:
I think you are right, the two other fields use it as well. I thought I did it without back then... Anyway, I think I'll have another look again tomorrow, because I'm curious *why* the state was necessary (should have added a comment when writing the other fields...)

Reply:
I just tried this and it worked fone:

```diff
diff --git a/indico/modules/events/editing/client/js/components/ExtensionList.jsx b/indico/modules/events/editing/client/js/components/ExtensionList.jsx
index 1b7fa73034..7005997148 100644
--- a/indico/modules/events/editing/client/js/components/ExtensionList.jsx
+++ b/indico/modules/events/editing/client/js/components/ExtensionList.jsx
@@ -4,10 +4,7 @@ import {Dropdown} from 'semantic-ui-react';
 import {Translate} from 'indico/react/i18n';

 export default function ExtensionList({value, disabled, onChange, onFocus, onBlur}) {
-  const [options, setOptions] = useState(value.map(x => ({text: x, value: x})));
-
   const handleChange = (e, {value: newValue}) => {
-    setOptions(newValue.map(x => ({text: x, value: x})));
     onChange(newValue);
     onFocus();
     onBlur();
@@ -15,7 +12,7 @@ export default function ExtensionList({value, disabled, onChange, onFocus, onBlu

   return (
     <Dropdown
-      options={options}
+      options={value.map(x => ({text: x, value: x}))}
       value={value}
       disabled={disabled}
       search
```